src/composeappmanager.cc

```diff
@@ -106,7 +106,10 @@ ComposeAppManager::ComposeAppManager(const PackageConfig& pconfig, const Bootloa
   if (!app_engine_) {
     auto docker_client{std::make_shared<Docker::DockerClient>()};
     auto registry_client{std::make_shared<Docker::RegistryClient>(http, cfg_.hub_auth_creds_endpoint)};
-    const auto compose_cmd{boost::filesystem::canonical(cfg_.compose_bin).string() + " "};
+    std::string compose_cmd{boost::filesystem::canonical(cfg_.compose_bin).string() + " "};
+    if (cfg_.compose_bin.filename().compare("docker") == 0) {
+      compose_cmd += "compose ";  // "make it `docker compose` command
+    }
```

Comment:
this will make it impossible to run a new aklite with an old platform. Its probably the easier fix, but we might also leave this as is and instead of check for "file exists" at startup, we check if `$cmd -h` works.

@ricardosalveti - you think we'll ever have someone wanting to run this and not also wanting to update their platform?

Reply:
Why impossible? a user still can use `docker-compose` by specifying it as a value of `docker_compose_bin` in his local env or additional toml snippet in meta-sub (as we agreed in the slack discussion).

Reply:
n/m i missed the "+ compose" part.

Reply:
Only for folks in dunfell that might not necessarily have docker compose v2 available, but as @mike-sul said, we can override the default via toml.